packages/server-wallet/src/protocols/close-channel.ts

```diff
@@ -53,15 +53,35 @@ function successfulWithdraw({app}: ProtocolState): boolean {
   return BN.eq(amountOwedToMe, amountTransferredToMe);
 }
 
+const isMyTurn = (ps: ProtocolState): boolean =>
+  !!ps.app.supported && (ps.app.supported.turnNum + 1) % 2 === ps.app.myIndex;
```

Comment:
Can we do something like this?
```suggestion
  !!ps.app.supported && (ps.app.supported.turnNum + ps.app.participants.length) % 2 === ps.app.myIndex;
```